test_schemas/en/test_thank_you_no_census_type_map.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
After the update I am half tempted to get rid of this schema, however we do need a test, and if I use a different schema in the test it potential conceals what is happening